sci-physics/rivet/rivet-3.1.6.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit python-single-r1 flag-o-matic autotools
+
+MY_PN="Rivet"
+MY_PF=${MY_PN}-${PV}
+
+DESCRIPTION="Rivet toolkit (Robust Independent Validation of Experiment and Theory)"
+HOMEPAGE="https://gitlab.com/hepcedar/rivet"
+SRC_URI="https://www.hepforge.org/archive/rivet/${MY_PF}.tar.gz"
+S=${WORKDIR}/${MY_PF}
+
+LICENSE="GPL-3+"
+SLOT="3"
+KEYWORDS="~amd64"
+IUSE="+hepmc3 hepmc2"
+REQUIRED_USE="
+	^^ ( hepmc3 hepmc2 )
+	${PYTHON_REQUIRED_USE}
+"
+
+RDEPEND="
+	>=sci-physics/yoda-1.9.5[python(-),${PYTHON_SINGLE_USEDEP}]
+	>=sci-physics/fastjet-3.4.0[plugins]
+	>=sci-physics/fastjet-contrib-1.048
+	hepmc2? ( sci-physics/hepmc:0=[-cm(-),gev(+)] )
```

Comment:
```suggestion
	hepmc2? ( sci-physics/hepmc:2=[-cm(-),gev(+)] )
```

Reply:
Thanks for spotting that. I wrote it before the slot 2 existed.
I have an similar ebuild for pythia with `hepmc2` and `hepmc3` flags. Should I PR that also to `::gentoo`? Though it invalidates/replaces the existing `hepmc` flag.

Reply:
> Should I PR that also to `::gentoo`? Though it invalidates/replaces the existing `hepmc` flag.

Sure! :+1: